src/server/conn_context.h

```diff
@@ -147,10 +147,7 @@ struct ConnectionState {
     const ConnectionContext* owner = nullptr;
   };
 
-  enum MCGetMask {
-    FETCH_CAS_VER = 1,
-    SET_EXPIRY = 2,
-  };
+  enum MCGetMask { FETCH_CAS_VER = 1 };
```

Comment:
These changes (within this file) are reverted from the previous PR. Because we now have a separate command for GAT, we no longer need to stash the expiry in the memcache flag, it is passed explicitly as part of the command.